README.md

```diff
@@ -11,3 +11,7 @@
 ## Category 1
 
 * [Placeholder link]()
+
+## How Do I connect to the Cluster?
```

Comment:
Hmm, I suppose we should decide whether we want these to be title-case or complete sentences. I could go either way.

I think I'm leaning towards complete sentences for now, so something like:
```markdown
## How do I connect to the cluster?
```

Any opinions or preferences?

Reply:
We can also go ahead and create a table-of-contents entry for this. I _believe_ for github-flavored markdown, this looks like:
```markdown
2. [How do I connect to the cluster?](#how-do-i-connect-to-the-cluster)
```

Reply:
Complete sentences are better